pkgs/development/python-modules/aiocron/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, croniter
+, tzlocal
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "aiocron";
+  version = "1.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-rX07m9iIl5NP/RiTf/I1q5+wE1goD9QOAUYf1fdjSL0=";
+  };
+
+  propagatedBuildInputs = [
+    croniter
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    tzlocal
+  ];
+
+  postPatch = ''
+    sed -i "/--cov/d" setup.cfg
+    sed -i "/--ignore/d" setup.cfg
+  '';
+
+  postInstall = ''
+    rm -rf $out/${python.sitePackages}/tests
+  '';
+
+  pythonImportsCheck = [ "aiocron" ];
+
+  meta = with lib; {
+    description = "Crontabs for asyncio";
+    homepage = "https://github.com/gawel/aiocron/";
+    license = licenses.mit;
```

Comment:
Missing platforms.

Reply:
I don't think this package has any platform-specific dependencies, it should use the platforms of the interpreter by default.

Reply:
I am almost sure that this will default to `lib.platforms.all` though, instead of just the platforms supported by Python itself.

Well, shouldn't be too much an issue though since Python will be a dependency too.

Reply:
The nixpkgs manual state "By default meta.platforms is set to the same value as the interpreter unless overridden otherwise." though I didn't dig through the source to confirm that. Most python packages don't have a platform either.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/cpython/default.nix#L505

```
nix-repl> legacyPackages.x86_64-linux.python3Packages.aiocron.meta.platforms
[ "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "i686-linux" "m68k-linux" "mipsel-linux" "powerpc64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "s390-linux" "s390x-linux" "x86_64-linux" "x86_64-darwin" "i686-darwin" "aarch64-darwin" "armv7a-darwin" ]
```

Yeah, looks like you're right.